test/e2e_flags.go

```diff
@@ -41,13 +40,12 @@ type EventingEnvironmentFlags struct {
 func initializeEventingFlags() *EventingEnvironmentFlags {
 	var f EventingEnvironmentFlags
 
-	repo := os.Getenv("DOCKER_REPO_OVERRIDE")
+	defaultRepo := os.Getenv("DOCKER_REPO_OVERRIDE")
```

Comment:
rename is needed?

Reply:
I think this is a merge from master: https://github.com/knative/eventing/pull/800